data/pycon-ca-2012/videos/how-you-can-contribute-to-python.json

```diff
@@ -1,31 +1,31 @@
 {
-  "id": 1578, 
-  "category": "PyCon CA 2012", 
-  "slug": "how-you-can-contribute-to-python", 
-  "title": "How you can contribute to Python", 
-  "summary": "Did you know that you could contribute to Python? From documentation\n\nwork to patch reviews to new features to blogging, there are a number of\n\nways you can give back to Python. This talk for beginners will walk you\n\nthrough these ways with concrete demonstrations.\n\n", 
-  "description": "", 
-  "quality_notes": "", 
-  "language": "English", 
-  "copyright_text": "", 
-  "thumbnail_url": "http://i2.ytimg.com/vi/i-dZ2pxJNFM/hqdefault.jpg", 
-  "duration": null, 
+  "id": 1578,
+  "category": "PyCon CA 2012",
+  "slug": "how-you-can-contribute-to-python",
+  "title": "How you can contribute to Python",
+  "summary": "Did you know that you could contribute to Python? From documentation\n\nwork to patch reviews to new features to blogging, there are a number of\n\nways you can give back to Python. This talk for beginners will walk you\n\nthrough these ways with concrete demonstrations.\n",
```

Comment:
ideally, extra newlines could be removed (they are in the original Markdown too)